packages/components/psammead-media-indicator/README.md

```diff
@@ -20,6 +20,15 @@ The `MediaIndicator` component provides a 'play' or 'audio' icon as well as a du
 | topStory      | boolean | No      | false   | true                         |
 | service | string | Yes | N/A | `'news'` |
 
+### Supported `type`s
```

Comment:
is this necessary?

Reply:
Perhaps not, but it'd be nice to display what options are available so that users are not forced to read the source code to discover what media types are supported by this component